ext/pgsql/config.m4

```diff
@@ -71,6 +71,7 @@ if test "$PHP_PGSQL" != "no"; then
   AC_CHECK_LIB(pq, PQchangePassword, AC_DEFINE(HAVE_PG_CHANGE_PASSWORD,1,[PostgreSQL 17 or later]))
   AC_CHECK_LIB(pq, PQsocketPoll, AC_DEFINE(HAVE_PG_SOCKET_POLL,1,[PostgreSQL 17 or later]))
   AC_CHECK_LIB(pq, PQsetChunkedRowsMode, AC_DEFINE(HAVE_PG_SET_CHUNKED_ROWS_SIZE,1,[PostgreSQL 17 or later]))
+  AC_CHECK_LIB(pq, PQclosePrepared, AC_DEFINE(HAVE_PG_CLOSE_STMT,1,[PostgreSQL 17 or later]))
```

Comment:
Here, now you just need to do rebase against the master branch and check it like this:

 ```m4
  PHP_CHECK_LIBRARY([pq], [PQclosePrepared],
    [AC_DEFINE([HAVE_PG_CLOSE_STMT], [1], [PostgreSQL 17 or later])],,
    [$PGSQL_LIBS])
```

Reply:
cool. I ll even wait your libpq bump merged.